src/test/java/core/basesyntax/service/transaction/TransactionStrategyTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.transaction;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionStrategyTest {
+    private static Map<FruitTransaction.Operation, TransactionHandler>
+            transactionHandlerMap;
+    private static TransactionStrategy transactionStrategy;
+
+    @Before
+    public void setUp() {
+        transactionHandlerMap = new HashMap<>();
+        transactionHandlerMap.put(FruitTransaction.Operation.BALANCE,
+                new BalanceTransactionHandler());
+        transactionHandlerMap.put(FruitTransaction.Operation.SUPPLY,
+                new SupplyTransactionHandler());
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void get_nullHandlerMap_notOk() {
+        transactionStrategy = new TransactionStrategy(null);
+        TransactionHandler transactionHandler =
+                transactionStrategy.get(FruitTransaction.Operation.BALANCE);
+    }
+
+    @Test
+    public void get_notExistKey_notOk() {
+        TransactionStrategy transactionStrategy =
+                new TransactionStrategy(transactionHandlerMap);
+        TransactionHandler transactionHandler =
+                transactionStrategy.get(FruitTransaction.Operation.RETURN);
+        assertNull(transactionHandler);
```

Comment:
```suggestion
        TransactionHandler transactionHandler =
                transactionStrategy.get(FruitTransaction.Operation.RETURN);
        assertNull(transactionHandler);
```
![image](https://user-images.githubusercontent.com/81781479/192300712-b12523ef-a124-4b95-90e8-741a18504853.png)

Reply:
not fixed